lib/ApplicationFeatures/RocksDBOptionFeature.cpp

```diff
@@ -255,7 +255,7 @@ void RocksDBOptionFeature::collectOptions(std::shared_ptr<ProgramOptions> option
       new Int64Parameter(&_blockCacheShardBits));
 
   options->addOption("--rocksdb.enforce-block-cache-size-limit",
-                     "if true, strictly enforces the block cache size limit",
+                     "if true, strictly enforces the block cache size limit (not supported)",
```

Comment:
Note: We do not re-generate examples for any version below 3.4, which means this change of the option description will not be reflected in the documentation automatically. When the PR is merged, `arangodb-ANY-examples` needs to be run manually (parameters: `3.3` in all three fields)